tensordict/nn/probabilistic.py

```diff
@@ -258,25 +261,28 @@ class ProbabilisticTensorDictModule(TensorDictModuleBase):
 
     """
 
-    SAMPLE_LOG_PROB_KEY = "sample_log_prob"
```

Comment:
maybe create a property that links to the new attribute for bc compatibility (+ deprec warning)

Reply:
i have tried to do this, is it what you were looking for?